opts/hosts.go

```diff
@@ -6,6 +6,8 @@ import (
 	"net/url"
 	"strconv"
 	"strings"
+
+	"github.com/docker/docker/daemon/network"
```

Comment:
Perhaps we should move the constant to somewhere in the client/api package; we're now importing the daemon package, which is not really ideal